fetool/styles/globals.css

```diff
@@ -1,26 +1,48 @@
-html,
-body {
-  padding: 0;
-  margin: 0;
-  font-family: -apple-system, BlinkMacSystemFont, Segoe UI, Roboto, Oxygen,
-    Ubuntu, Cantarell, Fira Sans, Droid Sans, Helvetica Neue, sans-serif;
+.content {
+  background-image: url("../assets/survey.png");
+  background-position: center;
+  background-repeat: no-repeat;
+  background-size: cover;
+  display: flex;
+  justify-content: center;
+  height: 70vh;
 }
 
-a {
-  color: inherit;
-  text-decoration: none;
+.heading{
+    display: flex;
+    justify-content: space-between;
```

Comment:
The class name and the css property doesn't seem to fit well. Can you clarify what is this class for?

Reply:
<header>
        <h1 className="heading">MSys</h1>

 </header>

This is for the header in the landing page (index.js)